File: eecs168/2020summer/2020.07.14.classes-part-2/Circle/Circle.cpp
//Implementation file for the Circle class
//Define all the methods for the Circle


//inclue the header file
#include "Circle.h"

//define the methods that were declared in
//the header
double Circle::area()
{
	double ans = 0;
	ans = 3.14*radius*radius;
	
	return(ans);	
}

File: eecs168/2022summer/2022.07.25.classes-part-5/circle.py
#circle.py

class Circle:
    def __init__(self, radius):
        self.radius = radius

    def area(self):
        ans = 3.14*self.radius**2
        return ans

    def __lt__(self, other):
        if self.radius < other.radius:
            return True
        else:
            return False

    def __str__(self):
        output_string = f'Circle with radius:{self.radius}'
        return output_string    


    def __repr__(self):
        string = f'Circle({self.radius})'
        return string
